docs/application/dotnet/samples/watch/overview.md

```diff
@@ -1,44 +1,50 @@
-			<p><a href="https://developer.tizen.org/development/sample/native/Watch/Ambient_Analog_Watch?langswitch=en" target="_blank">Native version</a></p>
-			<p><a href="https://developer.tizen.org/development/sample/web/Watch/Ambient_Watch?langswitch=en" target="_blank">Web version</a></p>
+			To use the ambient mode, you must enable it in the application settings. Launch <b>Settings</b> > <b>Watch faces and styles</b> > <b>Watch always on</b> > enable it. After screen timeout, the wearable device turns into ambient mode. In addition, the ambient mode activates only if you are wearing the watch on your wrist.</p>
+			<p>In addition, there are similar native and web sample applications.<br>
+			<a href="https://developer.tizen.org/development/sample/native/Watch/Ambient_Analog_Watch" target="_blank">Native version</a><br>
+			<a href="https://developer.tizen.org/development/sample/web/Watch/Ambient_Watch" target="_blank">Web version</a></p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wclassicwatch.png" width="180"/></td>
 			<td>
 			<p><strong>(W) ClassWatch</strong> [In progress]</p>
-			<p>The Classic Watch sample application demonstrates how to create a circular watch face, which consists of moving hands.</p>
-			<p><a href="https://developer.tizen.org/development/sample/native/Watch/Classic_Watch" target="_blank">Native version</a></p>
-			<p><a href="https://developer.tizen.org/development/sample/web/Watch/Classic_Watch" target="_blank">Web version</a></p>
+			<p>This sample application demonstrates how to create a circular watch face, which consists of moving hands.</p>
+			<p>In addition, there are similar native and web sample applications.<br>
+			<a href="https://developer.tizen.org/development/sample/native/Watch/Classic_Watch" target="_blank">Native version</a><br>
+			<a href="https://developer.tizen.org/development/sample/web/Watch/Classic_Watch" target="_blank">Web version</a></p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wchronograph.png" width="180"/></td>
 			<td>
 			<p><strong>(W) ChronographWatch</strong> [In progress]</p>
-			<p>The Chronograph Watch sample application demonstrates how to create a circular watch with continuously moving hands.
+			<p>This sample application demonstrates how to create a circular watch with continuously moving hands.
             This sample application includes the watch and stopwatch functionality.</p>
-			<p><a href="https://developer.tizen.org/development/sample/native/Watch/Chronograph_Watch" target="_blank">Native version</a></p>
-			<p><a href="https://developer.tizen.org/development/sample/web/Watch/Chronograph_Watch" target="_blank">Web version</a></p>
+			<p>In addition, there are similar native and web sample applications.<br>
+			<a href="https://developer.tizen.org/development/sample/native/Watch/Chronograph_Watch" target="_blank">Native version</a><br>
```

Comment:
In docs/application/dotnet/samples/appfw/overview.md, you listed the samples  when there was only one item. Then why no listing done here. Is there any specific reason?
Please verify and share your opinion.

Reply:
Yes, then we can use bulleted list here as well. Do you agree?

Reply:
Yes, I done.